app/partials/app.cjsx

```diff
@@ -11,7 +11,8 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
 
   @tokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy']
```

Comment:
What is the purpose of this? is it to ensure that we cannot post any other token to Geordi than one of these? Will we need to maintain a hardcoded list of valid tokens in future? That's probably not viable once we support logging on all / arbitrary projects.